concent_api/core/tests/test_unit_message_handlers.py

```diff
@@ -64,3 +67,22 @@ def test_that_messages_are_stored_with_correct_timestamps(self):
             self._parse_iso_date_to_timestamp(subtask.report_computed_task.timestamp.isoformat()),
             self._parse_iso_date_to_timestamp(self.report_computed_task_timestamp)
         )
+
+
+class TestForceSubtaskResultsResponse(ConcentIntegrationTestCase):
```

Comment:
I think this should be added to `AcceptOrRejectIntegrationTest` test case. Then you can send full request and test if response contains proper error code.